FirebaseMessaging/Tests/UnitTests/FIRMessagingTokenOperationsTest.m

```diff
@@ -438,4 +442,11 @@ - (void)stubInstallations {
   OCMStub([_mockInstallations authTokenWithCompletion:authTokenWithCompletionArg]);
 }
 
+- (void)stubHeartbeatInfo {
+  _mockHeartbeatInfo = OCMClassMock([FIRHeartbeatInfo class]);
+  NSInteger sdkAndGlobalHeartbeatRequestedCode = 3;
+  OCMStub([_mockHeartbeatInfo heartbeatCodeForTag:[OCMArg any]])
```

Comment:
We can validate the expected tag value passed to the method, e.g. 
```
OCMStub([_mockHeartbeatInfo heartbeatCodeForTag:@"expected_tag"])...
```

In this case the stub won't be applied to other argument values so the test will fail.